dsi/postgres/project_resources.go

```diff
@@ -544,29 +554,39 @@ func (d *Database) GetDefDocument(projectID, path, documentID string, filter map
 
 	// projectID
 	args = append(args, projectID)
-	filterString = append(filterString, fmt.Sprintf("project_id=$%d", index))
+	filterString = append(filterString, fmt.Sprintf("o.project_id=$%d", index))
 	index++
 
 	// document id
 	args = append(args, documentID)
-	filterString = append(filterString, fmt.Sprintf("id=$%d", index))
+	filterString = append(filterString, fmt.Sprintf("o.id=$%d", index))
 	index++
 
 	// valid sort/filter
 	validFields := map[string]bool{"*": true}
 
 	// filters
-	filterErr := d.mapToQuery(translatedFilters, validFields, &filterString, &args, &index)
+	filterErr := d.mapToQuery(translatedFilters, validFields, &filterString, &args, &index, "o")
 	if filterErr != nil {
 		return nil, dsiErrors.New(dsiErrors.UnknownError, filterErr)
 	}
 
-	queryFields := "id, creator, creator_type, created, data"
+	queryFields := "o.id, o.creator, o.creator_type, o.created, o.data"
+	joins := ""
+
+	relationIndex := 0
+	for key, r := range relations {
+		alias := r + strconv.Itoa(relationIndex)
```

Comment:
Why do we need the custom alias for "get" but just the relations index value in the "list" implementation?

Reply:
I changed it now, for a relation you need to pass the field and not the relation path